pylops_mpi/__init__.py

```diff
@@ -1,5 +1,6 @@
 from .DistributedArray import DistributedArray, Partition
 from .plotting.plotting import *
+from .basicoperators import MPIBlockDiag
```

Comment:
Why not `from .basicoperators import *` so it is already the correct form for later

Reply:
Done...